lib/serviceProvider.js

```diff
@@ -130,18 +141,23 @@ function doSoapRequest(url, method, soapAction, data, inputModel, outputModel, n
     body: soapBody,
     timeout: coreSettings.timeout || 15000
   };
+  var debug = {
+    startTime: new Date(),
+    request: {
+      method: method,
+      input: data,
+      request: soapRequest
+    }
+  };
+  if (coreSettings.debugSoap) coreSettings.logger(debug);
```

Comment:
Not sure if we should be doing this log here as it'll just get duplicated above but it'll be useful to have it just incase requests never make it back for some odd reason. What do others think?